### DISCUSSION TITLE
Is retry implemented?

### DISCUSSION THREAD

**Discussion by maelle** [eblondel/ows4R#79] [Q&A] [answered]
I searched for retry via GitHub but found nothing. Is it possible to use RETRY rather than e.g. GET? Wondering about this as the web service I'm using right now seems flaky. :sweat_smile: Thank you for your patience!

 cc @annakrystalli

**Comment by eblondel** [comment]
I'm not sure what you mean by implementing a RETRY here; If a web-service is down, there is nothing that ows4R can do... Retrying attempts to connect may be done, but that can be managed in user's code outside ows4R. what do you think?

**Comment by maelle** [comment]
I mean that maybe instead of GET/POST some methods could use https://httr.r-lib.org/reference/RETRY.html (btw see the interface for this in httr2 https://httr2.r-lib.org/reference/req_retry.html), letting the users pass the arguments pause_cap etc?

**Comment by eblondel** [comment, marked as answer]
It's not implemented, not sure if/when it will be implemented, but I will have a look.
